WTWSMS/decisions/realm_decisions.txt

```diff
@@ -684,7 +691,7 @@ decisions = {
 				narrative_event = { id = TOG.3100 }
 				create_character = {
 					random_traits = yes
-					religion = norse_pagan
+					religion = gernamic_pagan
```

Comment:
If the Varangarian Guard is formed, it would be ridiculous to give him the unused Norse Pagan religion. The Varangarian Captain would be Germanic Pagan instead. That was an oversight.